packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -85,6 +87,13 @@ export async function fetchHelper<T>(
                 `Error ${response.status} from the server`, response.status, response);
         }
 
+        if (!parse) {
+            return {
+                headers: response.headers,
+                content: IsoBuffer.from(await response.arrayBuffer()),
```

Comment:
Given that type is known upfront in such case, I'd rather have a dedication function that returns a buffer.
I.e. refactor code to have a "core" fetch funciton that does not parse anything and just returns Response, and then two functions on top of it - one that returns a buffer, and another one - generic, that does what today's fetchHelper does